packages/main/src/DayPicker.ts

```diff
@@ -110,20 +107,18 @@ type DayPickerNavigateEventDetail = {
 /**
  * Fired when the selected date(s) change
  * @public
```

Comment:
fix `namesTooLong` method. it has param and returns  tags which that are used wrong.

Reply:
done